README.md

```diff
@@ -413,9 +410,16 @@ pip install --upgrade "jax[cpu]"
 ```
 
 On Linux, it is often necessary to first update `pip` to a version that supports
-`manylinux2014` wheels. Also note that for Linux, we currently release wheels for `x86_64` architectures only, other architectures require building from source. Trying to pip install with other Linux architectures may lead to `jaxlib` not being installed alongside `jax`, although `jax` may successfully install (but fail at runtime).
-**These `pip` installations do not work with Windows, and may fail silently; see
-[above](#installation).**
+`manylinux2014` wheels. We currently release `jaxlib` wheels for the following
+operating systems and architectures:
+* Linux, x86-64
+* Mac (Intel)
+* Mac (ARM)
+* Windows, x86-64 (*experimental*)
+
+Other architectures require building from source. Trying to pip install on
+other architectures may lead to `jaxlib` not being installed alongside `jax`,
```

Comment:
```suggestion
Other platforms require building from source. Trying to pip install on
other platforms may lead to `jaxlib` not being installed alongside `jax`,
```

I'm guessing platform ≈ OS + architecture.

Reply:
FWIW I've been using "platform" to mean accelerator architecture, so I kinda like architecture to distinguish (not that it's unclear in this context) and I do think architecture is more precise.

Reply:
Being consistent with your terminology (which may be standard) SGTM, but my suggestion was just an attempt to make this text consistent with the text just above, especially "We currently release `jaxlib` wheels for the following operating systems and architectures". Based on that sentence I took architecture to mean something like hardware/ISA.

Reply:
Changed to "operating systems and architectures" in this paragraph.